### PR TITLE
Added error handling to assist debugging

### PR DESCRIPTION
**Pull request by narrigan**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by tt810 — APPROVED** [review]

**Comment by tt810** [comment]
could you resolve the conflicts please? Thanks

File: aws/vpc_flow_log_monitoring/lambda_function.py [hunk 1]
```diff
@@ -51,16 +51,17 @@ def process_message(message, tags, timestamp, node_ip):
...
+        process_log_status(log_status, detailed_tags, timestamp)
+        if log_status == 'NODATA':
+            return
+
+        process_action(action, detailed_tags, timestamp)
+        process_duration(start, end, detailed_tags, timestamp)
+        process_packets(packets, detailed_tags, timestamp)
+        process_bytes(_bytes, detailed_tags, timestamp)
+    except Exception, e:
+        print (e)
```
Comment (ajacquemot) [inline aws/vpc_flow_log_monitoring/lambda_function.py:64]: Can we have add a friendly message, something like `"Could not process message: {}".format(e)`

**Review by ajacquemot — APPROVED** [review]
Thanks for catching this, just left a nits but looks good, don't forget to rebase please 👍

**Comment by agocs** [comment]
Closing due to inactivity

**Pull request closed without merging**